backend/src/main/java/de/bund/digitalservice/ris/norms/application/port/output/SaveVerkuendungImportProcessPort.java

```diff
@@ -0,0 +1,59 @@
+package de.bund.digitalservice.ris.norms.application.port.output;
+
+import de.bund.digitalservice.ris.norms.domain.entity.VerkuendungImportProcess;
+import de.bund.digitalservice.ris.norms.domain.entity.VerkuendungImportProcessDetail;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ * Port for saving the progress information for the background processing
+ * of a new Verkündung.
+ */
+public interface SaveVerkuendungImportProcessPort {
+  /**
+   * Save the {@link VerkuendungImportProcess} object
+   *
+   * @param command containing the object of {@link VerkuendungImportProcess}
+   * @return The saved/updated {@link VerkuendungImportProcess}
+   */
+  VerkuendungImportProcess saveOrUpdateVerkuendungImportProcess(final Command command);
+
+  /**
+   * Parameter for saving/updating a {@link VerkuendungImportProcess} object
+   *
+   * @param id of the job
+   * @param status of the file import
+   * @param details of what went wrong
+   */
+  record Command(
+    UUID id,
+    VerkuendungImportProcess.Status status,
+    List<VerkuendungImportProcessDetail> details
+  ) {
+    /**
+     * Parameter for saving/updating a {@link VerkuendungImportProcess} object
+     *
+     * @param id of the job
+     * @param status of the file import
+     * @param details of what went wrong
+     */
+    public Command {
+      Objects.requireNonNull(id, "id cannot be null");
```

Comment:
i'm really unsure if the message here is really helpful or just one more thing we need to update if we ever want to change this again

Reply:
I agree with malte and we don't have something like this in any other command. Also this is just a fail-fast when calling the port in case of a null value was passed. But shouldn't we "catch" such case in the implementation of the output port?